src/site/content/en/_redirects.yaml

```diff
@@ -1,3 +1,16 @@
+# Master redirects file for web.dev. This mirrors the old DevSite file.
+# There are two types of redirects:
+#
+#  (1) Simple redirects, e.g., "from: /foo"
+#      - These don't match anything under this URL, like "/foo/bar"
```

Comment:
Maybe rephrase to something like:

These do not match **subpaths**. i.e., "from: /foo" will match "/foo" and "/foo/" but not "/foo/bar"